js/src/dapps/registry/Names/actions.js

```diff
@@ -77,30 +65,21 @@ export const drop = (name) => (dispatch, getState) => {
   const account = state.accounts.selected;
   const contract = state.contract;
   if (!contract || !account) return;
+  name = name.toLowerCase();
```

Comment:
Spacing before? (Also have an opportunity here to stop squashing the preceding line)